app/components/Machines.tsx

```diff
@@ -11,12 +11,23 @@ export default function Machines({ width }: { width: number }) {
   return (
     <Box maxW={width} backgroundColor="white">
       <Container maxW={adjustedWidth} bg="white" py="4rem">
-        <Heading color="black" textAlign="center" mb={4} pb={2}>
+        <Heading
+          fontSize={"48px"}
+          color="black"
```

Comment:
```suggestion
          color="brand.text"
```
Also, wanted to note here that I did a bit more styling on this section when I was working on Carousel Cards. Resolving merge conflicts will be painful lol